### DISCUSSION TITLE
How to handle error validation with jwtVerify() ?

### DISCUSSION THREAD

**Discussion by innovaweb-dev** [panva/jose#227] [Q&A]
How to handle error validation with jwtVerify() ?

```js
//...
try {
	await jwtVerify(refreshToken, refreshKey, {
		issuer: 'iss',
		audience: 'aud'
	}).then(error => {
		throw error
	})
} catch(error) {
	console.log(error);
}
```
That not works for me.

**Comment by panva** [comment]
@rifton007 you should probably look into how promises and async/await syntax works. Nothing to do with the library.

**Comment by innovaweb-dev** [comment]
Thank @panva, I know that not to do with the library. I just need help because i don't  know how to do that.
